grpc-circuitbreaker-utils/src/main/java/org/hypertrace/circuitbreaker/grpcutils/resilience/ResilienceCircuitBreakerRegistryProvider.java

```diff
@@ -0,0 +1,40 @@
+package org.hypertrace.circuitbreaker.grpcutils.resilience;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerConfig;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerRegistry;
+import jakarta.inject.Singleton;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.hypertrace.circuitbreaker.grpcutils.CircuitBreakerConfigParser;
+
+/** Utility class to provide Resilience4j CircuitBreakerRegistry */
+@Slf4j
+@Singleton
+public class ResilienceCircuitBreakerRegistryProvider {
+  private final Map<String, CircuitBreakerConfig> circuitBreakerConfigMap;
+
+  public ResilienceCircuitBreakerRegistryProvider(
+      Map<String, CircuitBreakerConfig> circuitBreakerConfigMap) {
+    this.circuitBreakerConfigMap = circuitBreakerConfigMap;
+  }
+
+  public CircuitBreakerRegistry getCircuitBreakerRegistry() {
+    CircuitBreakerRegistry circuitBreakerRegistry =
+        CircuitBreakerRegistry.of(
+            this.circuitBreakerConfigMap.get(CircuitBreakerConfigParser.DEFAULT_CONFIG_KEY));
+    circuitBreakerRegistry
+        .getEventPublisher()
+        .onEntryAdded(
+            entryAddedEvent -> {
+              CircuitBreaker addedCircuitBreaker = entryAddedEvent.getAddedEntry();
+              log.info("CircuitBreaker {} added", addedCircuitBreaker.getName());
```

Comment:
We really abuse info logs here - can this be debug? Or will it only happen once at startup?